scout/parse/panel.py

```diff
@@ -424,6 +424,19 @@ def get_omim_panel_genes(genemap2_lines, mim2gene_lines, alias_genes):
                 break
 
         if hgnc_id_info:
-            yield {"hgnc_id": hgnc_id_info["true"], "hgnc_symbol": hgnc_symbol}
+            if hgnc_id_info["true"]:
+                hgnc_id = hgnc_id_info["true"]
+            elif hgnc_id_info["ids"] and len(hgnc_id_info["ids"]) == 1:
```

Comment:
```suggestion
            elif len(hgnc_id_info.get("ids", ())) == 1:
```

Reply:
All aliases should have hgnc_id_info["ids"] in theory tho

Reply:
I guess one could also skip the first if and just use the `if len(hgnc_id_info.get("ids", ())) == 1`